site/components/pages/Pledge/lib/editPledgeSignature.ts

```diff
@@ -0,0 +1,2 @@
+export const editPledgeSignature = (nonce: string): string =>
+  `Sign to authenticate ownership and edit your Klima Infinity pledge 💚\n\nSignature nonce: ${nonce}`;
```

Comment:
Nice so this appears in metamask?! cool

Reply:
<img width="359" alt="image" src="https://user-images.githubusercontent.com/97446324/169025793-a1869c6b-9ca4-4273-a8cb-3ab273f4ee7d.png">
Yeah it's pretty cool :)
We can probably run the copy by someone to enhance the messaging to our users!